### ISSUE TITLE
Can't connect INT value from primitive node to Math expression node

### ISSUE THREAD

**Issue by lzv** [pythongosssss/ComfyUI-Custom-Scripts#196]
It highlighted, but doesn't connect.
![1](https://github.com/pythongosssss/ComfyUI-Custom-Scripts/assets/2552143/70dd7d9c-dc82-4cc3-a0cc-d0e4aac2775d)

**Comment by Zeerats** [comment]
I've got the same issue. Did you solve it?

**Comment by lzv** [comment]
> I've got the same issue. Did you solve it?

No. It looks like the repository is abandoned. I'm thinking of making my own nodes and removing this extension.
